### PR TITLE
Add #hs-api pages

### PR DESCRIPTION
**Pull request by alexsoble**
## What does this change? 

Add a v1 draft of the Eligibility APIs Initiative project site. 

## Federalist preview link

+ [Background](https://federalist-65bb532b-030a-4b62-a416-d7b74e4c0f2a.app.cloud.gov/preview/18f/portfolios/ars/hs-apis-subsite/projects/eligibility-apis/)

### CODE REVIEW

**Comment by AshleyO85** [comment]
procurement language looks good!

**Comment by alexsoble** [comment]
Thanks @AshleyO85 ! 😄 😄

File: _layouts/hs-apis-page.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  <section class="background-dark usa-section section-intro">
+    <div class="grid-container">
+      <div class="section-intro-header">
+          <h1 class="section-heading section-heading-alt">10x</h1>
+          <h2>Eligibility APIs Initiative</h2>
+      </div>
+    </div>
+  </section>
+
+  <section class="grid-container usa-section content-wide">
```
Comment (stvnrlly) [inline _layouts/hs-apis-page.html:21]: Would it make sense to give this some kind of element on top to make it clear that it's the side nav? I was thrown for a minute by seeing "background" and "background" next to each other before I figured it out.

Comment (alexsoble) [reply _layouts/hs-apis-page.html:34]: @stvnrlly Thanks for catching this, added code comments to make this clearer!

**Comment by alexsoble** [comment]
@Jacklynn Made the following assets changes that we discussed in standup: 

+ Added logo to header
+ Added gavel icon
+ Use static .png calculator image for now 

Let me know if the assets look in good shape for a v1 🚢 ! If we have any extra hours we can work on improvements like using the calculator GIF with a link to disable animations, and potentially adding more illustrations.

**Comment by Jacklynn** [comment]
@alexsoble I'd leave the alt tag empty since it's a decorative image `alt=""`

The h2 creates an extra space throwing the graphic and title proximity off 😔 
I'd either
1. Reduce the margin above the H2, this is at 8px (ideal, but the most custom)
Screen Shot 2020-09-15 at 11 38 20 AM
2. Put the graphic under the H2, There's slightly less margin after the title so I think this ok
Screen Shot 2020-09-15 at 11 38 53 AM
3. remove the graphic

Thank you so much!

**Comment by alexsoble** [comment]
@Jacklynn Removed alt tag and moved image below h2 in ff82b065f57c960ccd5a5ad631dcc29d5926a30a!

**Comment by Jacklynn** [comment]
@alexsoble I don't know if it's just my dev tools, but it looks like it just ends at `alt` and not the full `alt=""` -- does it do that for you or make a difference? I wonder if it looks less intentionally blank without the full attribute?

**Comment by alexsoble** [comment]
@Jacklynn — let me know what you think of the latest. We can keep improving this but I don't think it necessarily needs to be a blocker to 🚢  ing today

**Comment by Jacklynn** [comment]
looks good to me, thanks!

**Review by stvnrlly — APPROVED** [review]

**Pull request merged**